dm/tests/validator_basic/run.sh

```diff
@@ -34,6 +34,7 @@ function prepare_for_standalone_test() {
 }
 
 function trigger_checkpoint_flush() {
+	sleep 1.5
```

Comment:
the interval to flush validator checkpoint is 1s in this case right now, so need sleep before ddl(will trigger flush) for some time. below case is 1ms(will change to 1s in another fix pr) so sleep 0.1.